dsso_ros:
  ros__parameters:
    gammaFile: "/home/tp/src/dsso/EuRoC_MAV/pcalib.txt"
    vignetteFile: "/home/tp/src/dsso/EuRoC_MAV/vignette.png"
    calibFileL: "/home/tp/src/dsso/EuRoC_MAV/Tara1Left.txt"
    calibFileR: "/home/tp/src/dsso/EuRoC_MAV/Tara1Right.txt"